internal/pkg/sessions/cookie_store.go

```diff
@@ -68,10 +68,10 @@ func addPadding(secret string) string {
 }
 
 // CreateMiscreantCookieCipher creates a new miscreant cipher with the cookie secret
-func CreateMiscreantCookieCipher(cookieSecret string) func(s *CookieStore) error {
+func CreateMiscreantCookieCipher(cookieSecret []byte) func(s *CookieStore) error {
 	return func(s *CookieStore) error {
-		s.CookieSecret = cookieSecret
-		cipher, err := aead.NewMiscreantCipher(SecretBytes(s.CookieSecret))
+		s.CookieSecret = string(cookieSecret)
```

Comment:
Is this `CookieSecret` field on `CookieStore` actually needed/used?  From a quick grep through the codebase, it doesn't appear to be used.  Can we drop it?